drivers/pinmux/CMakeLists.txt

```diff
@@ -12,3 +12,4 @@ zephyr_sources_ifdef(CONFIG_PINMUX_SAM0            pinmux_sam0.c)
 zephyr_sources_ifdef(CONFIG_PINMUX_INTEL_S1000     pinmux_intel_s1000.c)
 zephyr_sources_ifdef(CONFIG_PINMUX_RV32M1          pinmux_rv32m1.c)
 zephyr_sources_ifdef(CONFIG_PINMUX_HSDK            pinmux_hsdk.c)
+zephyr_sources_ifdef(CONFIG_PINMUX_NPCX            pinmux_npcx.c)
```

Comment:
In the commit messsage, spelling for 'suport'

Reply:
Fixed